test/fixers/test_purs_tidy_fixer_callback.vader

```diff
@@ -0,0 +1,24 @@
+Before:
```

Comment:
Check https://github.com/dense-analysis/ale/blob/master/test/fixers/test_dhall_freeze_fixer_callback.vader for a simpler way to test a fixer using `ale#assert#SetUpFixerTest()`, `ale#assert#TearDownFixerTest()`, and `AssertFixer`.